Makefile

```diff
@@ -1,10 +1,11 @@
 .PHONY: build run lint test
+PORT ?= 5002
```

Comment:
It might be not the best practice to set this env var in the Makefile. How do we do this in Runner?

Reply:
Check my other comment: https://github.com/ONSdigital/eq-questionnaire-validator/pull/264#discussion_r2063054252